app/models/check/base.rb

```diff
@@ -0,0 +1,25 @@
+module Check
+  class Base < OkComputer::Check
```

Comment:
Consider refactoring. To me it looks like case no. 2 from https://stackoverflow.com/a/2947823/2987689. If `Base` is an abstract class, testing it looks a bit weird. It looks like there are some responsibilities that do not belong to this class.

Reply:
Moved to helper.